src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyCacheableRequest.java

```diff
@@ -121,9 +121,11 @@ void doNotModifiedResponse(
                                      routeId,
                                      replyId,
                                      authorization,
-                                     promiseNextPollRequest);
+                                     promiseNextPollRequest,
+                                     traceId);
```

Comment:
`traceId` should be earlier in the parameter list, between `replyId` and `authorization`.